src/native/corehost/build.cmd

```diff
@@ -5,7 +5,7 @@ setlocal
 :: Initialize the args that will be passed to cmake
 set "__sourceDir=%~dp0"
 :: remove trailing slash
-if %__sourceDir:~-1%==\ set "__ProjectDir=%__sourceDir:~0,-1%"
+if "%__sourceDir:~-1%"=="\" set "__sourceDir=%__sourceDir:~0,-1%"
```

Comment:
@akoeplinger, if we want to handle a pathological case when there was more than one backslash:

```suggestion
:removeBackslashes
if "%__sourceDir:~-1%"=="\" set "__sourceDir=%__sourceDir:~0,-1%" && goto removeBackslashes
```

Reply:
that shouldn't happen since __sourceDir is initialized to %~dp0 so it presumably can't have two backslashes

Reply:
Ah right, it's doesn't seem possible (if %~dp0 magic has done its job).